scratch/copy-mk.py

```diff
@@ -142,8 +144,8 @@ async def async_main(args) -> int:
                 "sync_time": sync_time,
                 "band": band,
                 "adc_sample_rate": adc_sample_rate,
-                "centre_frequency": BANDS[band].centre_frequency,  # TODO get from CAM
-                "antenna": label[:-1],
+                "centre_frequency": centre_frequency,
+                "antenna": observers[f"{label[:-1]}_observer"],
```

Comment:
Admittedly not yet clued up on the "uniqueness" of antennas. Why did you move away from simply `label[:-1]`?

Reply:
This gives the antenna *description* (i.e. its position).